helix-core/src/main/java/org/apache/helix/controller/stages/BestPossibleStateCalcStage.java

```diff
@@ -259,7 +259,7 @@ private boolean computeResourceBestPossibleState(ClusterEvent event, ResourceCon
         return checkBestPossibleStateCalculation(idealState);
       } catch (Exception e) {
         LogUtil
-            .logError(logger, _eventId, "Error computing assignment for resource " + resourceName + ". Skipping.", e);
+            .logError(logger, _eventId, "Error computing assignment for resource " + resourceName + ". Skipping.");
```

Comment:
should we print out the exception message, i.e, e.getMessage()? otherwise, do we know what error it encounters?

Reply:
I think if we print the exception message, the issue won't be resolved. The argue/issue was to avoid getting the exception message in this case.

Reply:
But here we catch all Exceptions, and swallow them, how do we know what type of exceptions we swallowed?

Reply:
You are right. But the problem is the discrepancy between default rebalance strategy and CRUSH based. As I understand from the issue, in the same situation we do not show exception in the default rebalance strategy. In my opinion, we should show exception either in both cases or in non of them.

Reply:
I fixed it in new PR.